Assets/Scripts/Player/PlayerMovement.cs

```diff
@@ -1,12 +1,15 @@
+using System;
 using UnityEngine;
 public class PlayerMovement : MonoBehaviour
 {
     private const string FLOORTAG = "Floor";
     private const string MFLOORTAG = "Mfloor";
```

Comment:
Separate words with `_`, `"FLOORTAG"` whould be `"FLOOR_TAG"`, also in `"MFLOORTAG"` what does `"M"` means, add a more descriptive name.